frontend/src/assets/datasuggest.js

```diff
@@ -14,53 +14,62 @@ export default [
     type: "vibe",
     picture: imgSad,
     alt: "smiley sad",
+    query: "with_genres=35&with_genres=28& with_genres=16",
   },
   {
     key: "happy",
     type: "vibe",
     picture: imgHappy,
     alt: "smiley happy",
+    query: "with_genres=10749&with_genres=16& with_genres=35",
   },
   {
     key: "thinking",
     type: "vibe",
     picture: imgThinking,
     alt: "smiley thinking",
+    query: "with_genres=99&with_genres=36& with_genres=10752",
   },
   {
     key: "angry",
     type: "vibe",
     picture: imgAngry,
     alt: "smiley angry",
+    query: "with_genres=80&with_genres=27& with_genres=9648&with_genres=53",
```

Comment:
- Never add spaces to an url
- I may be wrong but I think you overwrite the same variable (`with_genre`) over and over, it doesn't send the three categories. Have you tested it ?